GeoFire/GFCircleQuery.h

```diff
@@ -20,4 +20,9 @@
  */
 @property (atomic, readwrite) double radius;
 
+//private method
+- (id)initWithGeoFire:(GeoFire *)geoFire
+             location:(CLLocation *)location
+               radius:(double)radius;
+
```

Comment:
Any reason you chose to get rid of the private headers? This allows a clear separation of API and internal methods and makes it easier to document and understand what the API surface is.